syft/frameworks/torch/he/fv/context.py

```diff
@@ -12,14 +20,68 @@ class Context:
         rns_tool: A RNSTool class instance.
     """
 
-    def __init__(self, encryption_param):
+    def __init__(self, params):
 
-        self.param = encryption_param
+        # Validation of params provided for the encryption scheme.
+        self.validate(params)
+
+        self.param = params
+
+        self.rns_tool = RNSTool(params)
+
+    def validate(self, params):
+
+        # The number of coeff moduli is restricted to 62
+        if (
+            len(params.coeff_modulus) > COEFF_MOD_COUNT_MAX
+            or len(params.coeff_modulus) < COEFF_MOD_COUNT_MIN
+        ):
+            raise RuntimeError("Invalid coefficient modulus counts")
```

Comment:
Can you specify in the error message the range of the coeffecients allowed and how many was passed?